tests/languages/bash/environment_feature.test

```diff
@@ -0,0 +1,35 @@
+$foo
+$IFS
+IFS=a
+"$PS1"
+"${PS1}"
+
+----------------------------------------------------
+
+[
+	["variable", "$foo"],
+	["environment", "$IFS"],
+	["assign-left", [
+		["environment", "IFS"]
+	]],
+	["operator", ["="]],
+	"a\r\n",
+	["string", [
+		"\"",
+		["environment", "$PS1"],
+		"\""
+	]],
+	["string", [
+		"\"",
+		["variable", [
+			"${",
+			["environment", "PS1"],
+			"}"
+		]],
+		"\""
+	]]
+]
+
+----------------------------------------------------
+
+Checks for environment variables.

```

Comment:
It would be great to test for the whole list from `envVarList`, at least in one of their syntaxes (probably the simplest `$XXX` form).

Reply:
I was wondering if those tests with whole lists were overkill things or the standard way to go for Prism, haha. I'll do as you wish.

Reply:
They probably are a bit overkill, but they proved themselves useful in the past on a few occasions. 
And they are quick to write with a few "search and replace" action.

Reply:
You're perfectly right: creating the `builtin` test allowed me to notice that `cd` was in both functions and builtins, seconds ago. Haha.

```
$ type cd
cd is a shell builtin
```

→ Removed from functions.